dll/win32/shell32/shellmenu/CMenuFocusManager.cpp

```diff
@@ -324,6 +324,8 @@ LRESULT CMenuFocusManager::ProcessMouseMove(MSG* msg)
 
     POINT pt = msg->pt;
 
+    RECT itemRc = { 0 };
```

Comment:
```suggestion
    RECT rcItem;
```
Initialization can be omitted.

Reply:
I thought it would be safer to initialize it, I made ReactOS crash several times when I had a pointer that was uninitialized to NULL (it was unrelated to this thing). It might be ok to leave rcItem uninitialized here though.